Playlist_Genre_Rock.php

```diff
@@ -2,7 +2,7 @@
 <html lang = "eng">
```

Comment:
all genre playlists have the same changes; due to copy and pasting them all the same but change the query to suit the page